### PR TITLE
explained back-off delay 

### PR DESCRIPTION
**Pull request by KhushPatibandha**
worked on #18903 
explained back-off delay with primary behaviors and reasons to use back-off delay.

### CODE REVIEW

File: content/en/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 1]
```diff
@@ -156,6 +156,42 @@ in a Pod exit, the kubelet restarts them with an exponential back-off delay (10s
...
+the delay time exponentially, meaning that each subsequent delay is longer than the 
+previous one. The exponential increase gives the container more time to recover 
+before Kubernetes attempts to restart it again.
+
+- Maximum delay time: The back-off delay has a configurable maximum value that 
+sets an upper limit on the delay time. Once the delay time reaches this maximum 
+value, Kubernetes stops increasing the delay time and waits for the maximum 
+time before attempting to restart the container again.
+
+The reasons to use back-off delay are:
```
Comment (sftim) [inline content/en/docs/concepts/workloads/pods/pod-lifecycle.md:179]: Not sure this makes sense. The back off delay is imposed on you; there's no choice to a user of the Pod API about whether to use it.

**Review by sftim — COMMENTED** [review]
Thanks

We should frame this more as explaining what the kubelet does (ie: less emphasis on why you might want Kubernetes to have the back off before retrying).

**Review by Ritikaa96 — CHANGES_REQUESTED** [review]
Few more nits

**Review by Ritikaa96 — CHANGES_REQUESTED** [review]
Few more nits

**Comment by KhushPatibandha** [comment]
> Thanks
> 
> We should frame this more as explaining what the kubelet does (ie: less emphasis on why you might want Kubernetes to have the back off before retrying).

hey @sftim, should I put more emphasis on explaining how the kubelet handles "back off delay" or how kubelet assists us in "CrashLoopBackoff" in general?

**Comment by narharim** [comment]
Hey @sftim, I was looking for a detailed explanation for the back-delay off; if this PR gets merged, it will be helpful to me as well the community.
Also, I suggest adding a `CrashLoopBackOff` as Pod Phase. 
We have 5 phases currently in documentation `Pending`, `Running`, `Succeeded`,`Failed` and `Unknown.`
we can add `CrashLoopBackOff` after the `Failed` in documentation.

**Comment by tengqm** [comment]
> Hey @sftim, I was looking for a detailed explanation for the back-delay off; if this PR
> gets merged, it will be helpful to me as well the community.

Please avoid aiming at "get something merged". If you really want to "contribute",
think how to "get something improved/fixed".

> Also, I suggest adding a note of CrashLoopBackOff as Pod Phase.

Before doing that, you have to understand the definition of "Phase".
Are you sure that `CrashLoopBackOff` is defined as a Pod phase?

**Comment by narharim** [comment]
> Please avoid aiming at "get something merged". If you really want to "contribute", think how to "get something improved/fixed".
> 
I apologize if my previous comment sounded like I was solely focused on getting something merged. I understand that is not about getting something merged but about positive impact on the community. Thank you for your valuable feedback
> 
> Before doing that, you have to understand the definition of "Phase". Are you sure that `CrashLoopBackOff` is defined as a  Pod phase?

Thank you for highlight this as well.You are correct. I should have double-checked the difference between "Pod status" and "Pod phase" before making the suggestion.

**Comment by Ritikaa96** [comment]
Hi @KhushPatibandha are you willing to address the reviews suggested ? are you still working on this PR?

**Comment by tengqm** [comment]
@KhushPatibandha Still working on this one? If you don't have cycles to push this forward, is it okay for someone else to take over this and get the problem fixed?

**Comment by Ritikaa96** [comment]
Hi @KhushPatibandha , Please address the changes if you are still working or you can hand this issue to someone else who has bandwidth to fix this.

**Comment by kbhawkey** [comment]
👋 @KhushPatibandha .
I am closing this pull request as it has been open for some time without activity.
You can reopen your pull request and continue to work on the changes.
/close

**Pull request closed without merging**